app/api/main.py

```
@@ -0,0 +1,282 @@
+
+    return app
+
+
+# Expose a default app instance for simple `uvicorn app.api.main:app` usage.
+# The provided run command in README uses `create_app` as a factory; exposing
+# `app` as well improves interoperability.
+try:
+    app = create_app()
+except Exception:
+    # If settings validation fails at import time, bubble up a clear message.
+    raise
```

Ruff (0.13.1)

Remove exception handler; error is immediately re-raised

(TRY203)